packages/fabric8-ui/src/polyfills.browser.ts

```diff
@@ -28,12 +28,12 @@ import 'core-js/es6/weak-set';
 import 'core-js/es7/reflect';
 import 'zone.js/dist/zone';
 
+require('zone.js/dist/long-stack-trace-zone');
```

Comment:
This should be inside the if statement still. Just add a comment to disable the rule.